docs/fsharp/using-fsharp-on-azure/queue-storage.md

```diff
@@ -124,40 +110,40 @@ software failure, another instance of your code can get the same message
 and try again. Your code calls `DeleteMessage` right after the message
 has been processed.
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L75-L76)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L61-L62)]
 
 ## Use Async workflows with common Queue storage APIs
 
 This example shows how to use an async workflow with common Queue storage APIs.
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L82-L91)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L68-L77)]
 
 ## Additional options for de-queuing messages
 
 There are two ways you can customize message retrieval from a queue.
 First, you can get a batch of messages (up to 32). Second, you can set a
 longer or shorter invisibility timeout, allowing your code more or less
 time to fully process each message. The following code example uses
-`GetMessages` to get 20 messages in one call and then processes
+`ReceiveMessages` to get 20 messages in one call and then processes
```

Comment:
Is this still true that it will get 20 messages?

Reply:
Looks that way from the code snippet.

Reply:
Yes, it was set [here](https://github.com/zedy-wj/docs/blob/queue-storage/samples/snippets/fsharp/azure/queue-storage.fsx#L88).